test/unit/test_keyring_multi.py

```diff
@@ -108,7 +123,12 @@ def test_keyring_with_children_but_no_generator():
             )
         ]
     )
-    assert isinstance(test_multi_keyring.children, list)
```

Comment:
See comments on previous test re matching attributes of keyring vs matching identity.